### PR TITLE
Update migrations.md

### PR DESCRIPTION
**Pull request by redeemefy**
Add more detail instructions if the command is used in combination with `package.json/scripts`

### CODE REVIEW

File: versioned_docs/version-6.x.x/other-topics/migrations.md [hunk 1]
```diff
@@ -562,6 +562,22 @@ As an alternative to the `--config` option with configuration files defining you
 npx sequelize-cli db:migrate --url 'mysql://root:password@mysql_host.com/database_name'
 ```
 
+If utilizing `package.json/scripts` commands make sure to use the extra `--` in your cammand.
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/migrations.md:565]:
```suggestion
If utilizing `package.json/scripts` commands make sure to use the extra `--` in your command when using flags.
```

**Review by ephys — DISMISSED** [review]

**Comment by ephys** [comment]
This should also be documented in migrations.md for v7 :)

**Comment by redeemefy** [comment]
> Thanks for the PR :)
> 
> Can you replicate your changes in `docs/other-topics/migrations.md` too, for v7?

[PR for updating v7](https://github.com/sequelize/website/pull/190)

**Comment by ephys** [comment]
Thanks :)

**Pull request merged**